src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -6497,6 +6497,95 @@ namespace Microsoft.FSharp.Core
                   interface IEnumerable with 
                       member x.GetEnumerator() = (gen() :> IEnumerator) }
 
+            [<NoEquality; NoComparison>]
+            type VariableStepIntegralRangeState<'a> = {
+                mutable Started  : bool
+                mutable Complete : bool
+                mutable Current  : 'a
+            }
+            let inline variableStepIntegralRange n step m =
+                if step = LanguagePrimitives.GenericZero
+                then invalidArg "step" (SR.GetString(SR.stepCannotBeZero));
+                else
```

Comment:
i think the `else` it's not needed, because `invalidArg` raise an exception ( so a `if then` it's enough, like [there](https://github.com/manofstick/visualfsharp/blob/d5d4c49a30d924ff5388f11eb82cc770e8ca5839/src/fsharp/FSharp.Core/prim-types.fs#L6632)
Less indentation